include/kframework/schedule.md

```diff
@@ -0,0 +1,345 @@
+/// TODO DOCS
+
+### Default Schedule
+
+```k
+requires "data.md"
+module SCHEDULE
+    imports EVM-DATA
+
+    syntax Schedule ::= "DEFAULT" [klabel(DEFAULT_EVM), symbol, smtlib(schedule_DEFAULT)]
```

Comment:
I think it is necessary to have a separate `SCHEDULE` module, as the functions in `gas.md` are dependant on `Schedule`. I am unsure how much should be included here from `evm.md`, there is more that I left behind.

Reply:
Yes, this makes sense. Anything that is purely functional should probably be moved here.